### ISSUE TITLE
operator-whitespace returns a false positive on line-broken operators

### ISSUE THREAD

**Issue by ritave** [duaraghav8/Ethlint#175]
Example:

```solidity
    if (
      appWasMade(_listingAddress) &&
      listings[_listingAddress].applicationExpiry < now &&
      !getListingIsWhitelisted(_listingAddress) &&
      (challengeID == NO_CHALLENGE || challenges[challengeID].resolved == true)
    ) {
      return true;
    }
```

Errors out on `&&` on each line with `Single space should be either on both sides of '&&' or not at all.`

**Comment by duaraghav8** [comment]
Could you please tell me your Solium version?
`solium -V`

**Comment by ritave** [comment]
```
$ solium -V
1.1.5
```

Happy to help

**Comment by ritave** [comment]
Additional info (red herring?) - might be due to mixed line endings in file.

**Comment by PaulRBerg** [comment]
You can temporarily silence the error by prepending the statement with `// solium-disable-next-line operator-whitespace`.

**Comment by duaraghav8** [comment]
Resolved in `v1.2.2`.
Sorry this took way too long, but its done now!

**Comment by Siegrift** [comment]
I think I found a code for which the fix doesn't work properly.

This is the line which reports `operator-whitespace` warning: https://github.com/vacuumlabs/name-bazaar/blob/not-working-op-whitespace/resources/public/contracts/src/AuctionOffering.sol#L159

`npm run ethlint --version`  = 6.14.4
